modules/alloy/src/main/ts/ephox/alloy/menu/build/ItemType.ts

```diff
@@ -37,7 +45,13 @@ const builder = (detail: NormalItemDetail): AlloySpec => ({
         aria: {
           mode: 'checked'
         },
-        ...tConfig
+        ...Obj.filter(tConfig as { [K in keyof TogglingConfigSpec]: TogglingConfigSpec[K] }, (_value, name) => name !== 'exclusive'),
```

Comment:
This is a bit hard to unpack. Can we move it above `builder`?

Reply:
Have split it out and added a comment... hopefully that's what you meant.